### PR TITLE
New global parameters UI widget

### PR DESCRIPTION
**Pull request by alrusdi**
It looks like this https://alrusdi.ru/static/globs2.png

### CODE REVIEW

File: src/components/GlobalParameters.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+export const GlobalParameters = Vue.component("globs", {
+    props: ["oceans_count", "oxygen_level", "temperature"],
+    data: function () {
+        return {}
+    },
+    "methods": {
+        "getValuesRange": function (for_param: string): Array<string> {
+            let values: Array<string> = []; 
+            if (for_param == "oxygen") {
+                for (let i = 14; i>=0; i--) {
```
Comment (vincentneko) [inline src/components/GlobalParameters.ts]: I think it's better to use constants defined in constants.ts file, like MAX_OXYGEN_LEVEL for instance

File: src/components/GlobalParameters.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                    values.push(i.toString())
+                }
+            } else {
+                for (let i = 8; i>=-30; i-=2) {
```
Comment (vincentneko) [inline src/components/GlobalParameters.ts]: Same here

File: src/components/GlobalParameters.ts [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+            <div class="globs_columns_cont">
+                <div class="globs_column globs_column--temperature">
+                    <div class="globs_column_scale globs_column_scale--temperature" :style="getScaleCSS('temperature')"></div>
+                    <div class="globs_value" v-for="val in getValuesRange('temperature')">{{ val }}</div>
+                </div><div class="globs_column globs_column--oxygen">
+                    <div class="globs_column_scale globs_column_scale--oxygen" :style="getScaleCSS('oxygen')"></div><div class="globs_value" v-for="val in getValuesRange('oxygen')">{{ val }}</div>
+                </div>
+            </div>
+            <div class="globs_oceans">
+                <div class="globs_oceans_count">{{ oceans_count }}/9</div>
```
Comment (vincentneko) [inline src/components/GlobalParameters.ts]: And here

File: src/components/GlobalParameters.ts [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+        "getScaleCSS": function (for_param: string): string {
+            let ret = "";
+            const val:string = (for_param == "oxygen") ? this.oxygen_level : this.temperature;
+            const num_val = parseInt(val);
+            if (for_param == "temperature") {
+                const diff = Math.abs(-32 - num_val) / 2;
+                const height = diff * 17;
+                const margin_top = 322 + 17 - height;
+                ret = "margin-top: " + margin_top.toString() + "px; height: " + height + "px;";
+                console.log(ret);
```
Comment (vincentneko) [inline src/components/GlobalParameters.ts]: Please remove this line

**Comment by alrusdi** [comment]
Thank you for review! Tried to fix all described issues. Can you please check it again?

**Pull request merged**